rpcs3/Emu/Io/usio.h

```diff
@@ -21,6 +21,10 @@ class usb_device_usio : public usb_device_emulated
 	void usio_read(u8 channel, u16 reg, u16 size);
 
 private:
+	bool test_on = false;
+	bool last_test = false;
+	bool last_coin = false;
```

Comment:
I think these should be named test_key_pressed and coin_key_pressed

Reply:
Done.